code/modules/ninja/energy_katana.dm

```diff
@@ -0,0 +1,10 @@
+
+
+/obj/item/weapon/katana/energy
+	name = "energy katana"
+	desc = "a katana infused with a strong energy"
+	icon_state = "energy_katana"
+	item_state = "energy_katana"
+	force = 40
+	throwforce = 20
+	slot_flags = SLOT_BELT
```

Comment:
add backslot too pls
everyone knows true space ninjas have them on the back

![](http://www.unbox.ph/wp-content/uploads/2013/09/IMG_8689.jpg)